frontend/sw360-portlet/src/main/resources/content/Language_ja.properties

```diff
@@ -209,14 +209,16 @@ ClearingRequestPriority-CRITICAL=クリアリング時間が1週以内.
 ClearingRequestPriority-HIGH=クリアリング時間が1週間以上2週間以内.
 ClearingRequestPriority-MEDIUM=クリアリング時間が2週間以上4週間以内.
 ClearingRequestPriority-LOW=クリアリング時間が4週間以上.
-ClearingRequestState=New: 保留されていたクリアリングリクエストがクリアリングチームによって承認.&#10A クリアリングに必要な成果物のSanityCheckはクリアリングリクエスト承認前に行う &#10Accepted: クリアリングリクエストはSanityCheckが完了しクリアリング受領. &#10Rejected:クリアリングリクエストはSanityCheckに失敗. クリアリングチームに連絡してください. (Rejecting resets the CR priority) &#10In Queue:クリアリングリクエストが次のリクエストに入りました． &#10In Progress: クリアリングリクエストはクリアリングチームに確認されています &#10Closed: クリアリングリクエスト完了 (Closing resets the CR priority) &#10Awaiting Response: Clearing Team is waiting for input/action from the Requesting User.
+ClearingRequestState=New: 保留されていたクリアリングリクエストがクリアリングチームによって承認.&#10A クリアリングに必要な成果物のSanityCheckはクリアリングリクエスト承認前に行う &#10Accepted: クリアリングリクエストはSanityCheckが完了しクリアリング受領. &#10Rejected:クリアリングリクエストはSanityCheckに失敗. クリアリングチームに連絡してください. (Rejecting resets the CR priority) &#10In Queue:クリアリングリクエストが次のリクエストに入りました． &#10In Progress: クリアリングリクエストはクリアリングチームに確認されています &#10Closed: クリアリングリクエスト完了 (Closing resets the CR priority) &#10Pending Input: Clearing Team is waiting for input/action from the Requesting User.
 ClearingRequestState-NEW=保留されていたクリアリングリクエストがクリアリングチームによって承認.&#10A クリアリングに必要な成果物のSanityCheckはクリアリングリクエスト承認前に行う.
 ClearingRequestState-ACCEPTED=クリアリングリクエストはSanityCheckが完了しクリアリング受領.
 ClearingRequestState-REJECTED=クリアリングリクエストはSanityCheckに失敗. クリアリングチームに連絡してください. (Rejecting resets the CR priority)
 ClearingRequestState-IN_QUEUE=クリアリングリクエストが次のリクエストに入りました．
 ClearingRequestState-IN_PROGRESS=クリアリングリクエストはクリアリングチームに確認されています．
 ClearingRequestState-CLOSED=クリアリングリクエスト完了． (Closing resets the CR priority)
 ClearingRequestState-AWAITING_RESPONSE=クリアリングチームはユーザーからの対応待ちです．
+ClearingRequestState-PENDING_INPUT=Clearing Team is waiting for input/action from the Requesting User.
+ClearingRequestState-SANITY_CHECK = A sanity check for the artifacts required for Clearing will be done prior to Accepting a CR
```

Comment:
ClearingRequestState-SANITY_CHECK = Prior to accepting a CR, a sanity check for the artifacts required for Clearing will be performed.